core/loadpoint.go

```diff
@@ -1156,7 +1156,7 @@ func (lp *Loadpoint) pvMaxCurrent(mode api.ChargeMode, sitePower float64, batter
 
 	if mode == api.ModePV && lp.enabled && targetCurrent < minCurrent {
 		// kick off disable sequence
-		if sitePower >= lp.Disable.Threshold && lp.phaseTimer.IsZero() {
+		if sitePower-minCurrent*float64(activePhases-1) >= lp.Disable.Threshold || (sitePower >= lp.Disable.Threshold && lp.phaseTimer.IsZero()) {
```

Comment:
Grundsätzliche Frage: Wieso wird für die Berechnung `activePhases-1` verwendet? Wenn ich gerade mit 3 Phasen lade, würde dann ja für 2 Phasen geprüft ob es danach noch reicht. Da die Boxen aber sowieso von 3 auf 1 schalten, sollte hier `-2` verwendet werden.

Reply:
Wird es.
- (3-1) = -2

Reply:
Das in eine Nebenrechnung rauszuziehen und auch auf 0 zu prüfen erhöht die Klarheit.

Reply:
Hab's in eine Nebenrechnung rausgezogen. Prüfung auf 0 ist nicht notwendig.